server/src/main/java/org/spine3/server/entity/AbstractEntity.java

```diff
@@ -157,12 +160,126 @@ private static IllegalStateException noSuchConstructor(String entityClass, Strin
     }
 
     /**
+     * Obtains constructor for the passed aggregate part class.
```

Comment:
Article is added. I described it in the next paragraph. Should I expand somehow it?